src/notation/view/notationpaintview.cpp

```diff
@@ -26,6 +26,11 @@
 using namespace mu::notation;
 using namespace mu::framework;
 
+static constexpr qreal MIN_SCROLL_SIZE = 0.2;
+static constexpr qreal MAX_SCROLL_SIZE = 1.0;
+
+static constexpr qreal CANVAS_SIDE_MARGIN = 8000;
```

Comment:
Please, use some proportional value instead of this magic number (that depends on the current value of width/height). You also should use guiScale() when you will calculate this value

Reply:
I will fix this issue later